x-pack/plugins/cases/public/components/all_cases/selector_modal/use_cases_add_to_existing_case_modal.tsx

```diff
@@ -126,22 +137,21 @@ export const useCasesAddToExistingCaseModal = (props: AddToExistingCaseModalProp
       dispatch({
         type: CasesContextStoreActionsList.OPEN_ADD_TO_CASE_MODAL,
         payload: {
-          ...props,
```

Comment:
The payload accepts the properties of `AllCasesSelectorModalProps`. Could you please add in the payload `onCreateCaseClicked: props.onCreateCaseClicked`?